tdb/vidrl_upload.py

```diff
@@ -24,6 +24,20 @@
 # This is based on the vaccine composition for the Southern Hemisphere
 # because all human pooled sera should be from Australia
 VACCINE_MAPPING = {
+    "2023": {
```

Comment:
Ended up needing to add 2023 vaccine mapping because some of the 2024 files included human sera references from 2023.

Reply:
Based on the date of the XLS file and other context, do you feel confident that the date in the spreadsheet(s) was intended to be "2023" and not "2024"? Do you have an example spreadsheet with 2023 human pool data that I could check out?

Reply:
Good point! I'll dig up some examples.

Reply:
Agreed, that sounds right. Thank you for checking!